Tasks/AppCenterDistributeV3/task.json

```diff
@@ -63,7 +63,8 @@
             "defaultValue": "Apple",
             "groupName": "symbols",
             "options": {
-                "Apple": "Apple"
+                "Apple": "Apple",
+                "AndroidProguard": "Android"
```

Comment:
Why do we name it "AndroidProguard"? What are other possible variants of "Android" in the future? And when we add those, what would the display name be?

Reply:
`AndroidProguard` is the value that's used for the symbol type in App Center Diagnostics. This currently keeps a direct relationship between the values.
We will most likely not be adding different symbol types to the selection, but additional supported symbol types within the Android selection. This will require more changes to the internals of the symbol upload which currently relies on that fact that there is only one symbol/artifact field to upload.
I can make the name be more generic "Android" and introduce a mapping between "Android" on the task side and "AndroidProguard" on the symbol upload side.

Reply:
Updated this part to have the input parameter name be only `Android`, this allows it to be extensible without necessarily breaking the API once we introduce other symbol support for Android (native, ...) and is then internally converted to the right symbol type for upload to App Center Diagnostics.